### ISSUE TITLE
bundle-sfx arithmetics/--globals broken since some patch version

### ISSUE THREAD

**Issue by neothemachine** [jspm/jspm-cli#1159]
I used the following successfully previously:
```
jspm bundle-sfx leaflet-coverage/classic - leaflet leaflet-coverage.src.js --globals "{leaflet:'L'}" --format global --skip-source-maps
```
I'm not sure which jspm version that was, something between 0.16.2 and 0.16.9. I'm now at 0.16.10 and noticed it doesn't work anymore. When I try to use the bundle I get the following:
```
Uncaught Module 81 not present.
load @ leaflet-coverage.src.js:365
linkDeclarativeModule @ leaflet-coverage.src.js:200
linkDeclarativeModule @ leaflet-coverage.src.js:204
link @ leaflet-coverage.src.js:127
load @ leaflet-coverage.src.js:369
(anonymous function) @ leaflet-coverage.src.js:425
(anonymous function) @ leaflet-coverage.src.js:8619
loader @ leaflet-coverage.src.js:387
(anonymous function) @ leaflet-coverage.src.js:8618
```

When I look into the generated source, I see `"github:Leaflet/Leaflet@0.7.3"` at some point, and not `"leaflet"` as I aliased it and how I import it. I think this is the problem.

You can check it for yourself by cloning https://github.com/Reading-eScience-Centre/leaflet-coverage and then doing `npm install` and putting the generated `leaflet-coverage.src.js` into a folder with an `index.html` which has:
```html
<!DOCTYPE html>
<html>
<body>
<script src="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
<script src="leaflet-coverage.src.js"></script>
<script>
    console.log(L);
    console.log(L.coverage);
</script> 
</body>
</html>
```

**Comment by neothemachine** [comment]
So, I just got feedback from a colleague that it works in 0.16.2 and breaks in 0.16.3. That should help I hope.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/e49ef34994e9574b770d55c639c4596d003d8ee0.

**Comment by guybedford** [comment]
Released in 0.16.12.
